db/seeds.rb

```diff
@@ -6,7 +6,7 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-# names = ['Juan', 'Pekka', 'Luula', 'Aarnio', 'Aarto']
+names = ['Juan', 'Pekka', 'Luula', 'Aarnio', 'Aarto']
 5.times do |i|
 	user = User.create(name: names[i], photo: "https://randomuser.me/api/portraits/men/jpg", bio: 'Lots of things to say in here')
```

Comment:
**Optonal Changes**
- [ ] The seed currently tries to submit a null value. Please let us try to fix it or simply remove it and add test data using `rails c`